loop/views.py

```diff
@@ -53,7 +54,7 @@ def dashboard(request):
 
 def filter_data(request):
     language = request.GET.get('language')
-    aggregators = LoopUser.objects.all().values('user__id', 'name', 'name_en')
+    aggregators = LoopUser.objects.filter(role=ROLE_AGGREGATOR).values('user__id', 'name', 'name_en')
```

Comment:
Same thing needs to be handled at other places as well. E.g.  LINE 177 in function total_static_data(request)